mocks/handlers/accountHandler.js

```diff
@@ -48,6 +48,7 @@ export const accountHandler = graphql.query('getAccount', (req, res, ctx) => {
           feeFrozen: '198480993716691',
           formattedFeeFrozen: '19,848.0993 DOT',
         },
+        subAccounts: [],
```

Comment:
Don't we need any mock data for the sub accounts??

Reply:
I have tried to mock the data, but I think the data I mocked is wrong, and unable to find a relevant example account. Is it possible for you to provide a mock for this?

Reply:
```
{
  substrateChainAccount(address: "13KJ3t8w1CKMkXCmZ6s3VwdWo4h747kXE88ZNh6rCBTvojmM") {
    address
    subAccounts {
      address
    }
  }
}
```

This account has sub accounts. You can get the mock from here. Add it as an external account in the app and you can get the data.